spec/models/ems_folder_spec.rb

```diff
@@ -1,49 +1,19 @@
 describe EmsFolder do
   context "#hidden?" do
-    it "when not VMware" do
+    it "when not hidden" do
       folder = FactoryGirl.build(:ems_folder, :name                  => "vm",
+                                              :hidden                => false,
                                               :ext_management_system => FactoryGirl.build(:ems_openstack)
                                 )
       expect(folder).to_not be_hidden
     end
```

Comment:
I think both of these tests can go away completely because at this point you're just testing Rails' attribute methods on a boolean column.  You can do this is a follow up PR.